go.mod

```diff
@@ -49,6 +49,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.0
 	github.com/psampaz/go-mod-outdated v0.8.0
+	github.com/rjeczalik/notify v0.9.2
```

Comment:
Do we wanna switch to this dependency completely instead of `github.com/fsnotify/fsnotify`?

Reply:
I found there were test failures with it and I want to jam this through first.